packages/dialtone-vue3/recipes/conversation_view/emoji_row/emoji_row.vue

```diff
@@ -159,7 +159,10 @@ export default {
   }
 
   &__reaction-number {
-    font: var(--dt-typography-body-small);
+    // font: var(--dt-typography-body-sm); // replace with this once Style Dictionary transform resolved
```

Comment:
I think this is part of what needs to be replaced with the last changes from @braddialpad 
The same with the other cases that have `// replace with this once Style Dictionary transform resolved`

Reply:
I got it.